asserts/database.go

```diff
@@ -93,14 +94,41 @@ func (nbs nullBackstore) SequenceMemberAfter(t *AssertionType, kp []string, afte
 	return nil, &NotFoundError{Type: t}
 }
 
+// keyNotFoundError is returned when the key with a given ID cannot be found.
+type keyNotFoundError struct {
+	keyID string
+	where string
+}
+
+func (e *keyNotFoundError) Error() string {
+	where := ""
+	if e.where != "" {
+		where = fmt.Sprintf(" %s", e.where)
+	}
+	return fmt.Sprintf("cannot find key %q%s", e.keyID, where)
```

Comment:
s/key/key pair/

Reply:
Also updated the rest of the code to use key pair and drop the name/ID from the message, while adding key names in places where more error context was needed.